test/src/tests-store.ts

```diff
@@ -0,0 +1,139 @@
+import { expect } from "chai";
+import sinon from 'sinon';
+import { SDKInitConfig } from "../../src/sdkRuntimeModels";
+import Store, { IStore } from "../../src/store";
+import { MPConfig, apiKey } from './config';
+
+describe('Store', ()=> {
+    const now = new Date();
+    let sandbox;
+    let clock;
+
+    const sampleConfig = {
+        appName: 'Store Test',
+        appVersion: '1.x',
+        package: 'com.mparticle.test',
+    } as SDKInitConfig;
+
+    beforeEach(function() {
+        sandbox = sinon.createSandbox();
+        clock = sinon.useFakeTimers(now.getTime());
+        // MP Instance is just used because Store requires an mParticle instance
+        window.mParticle.init(apiKey, window.mParticle.config);
+    });
+
+    afterEach(function() {
+        window.mParticle._resetForTests(MPConfig);
+        sandbox.restore();
+        clock.restore();
+    });
+
+	it('should initialize with defaults', () => {
```

Comment:
indentation is wonky here.  remove 1 tab.  hard to tell if my suggestion is doing that properly, so I'll leave a comment instead of a suggestion